girder/cli/shell.py

```diff
@@ -42,15 +44,24 @@ def _launchShell(context):
         return code.interact(banner=header, local=context)
 
 
-@click.command('shell', short_help='Run a Girder shell.', help='Run a Girder shell.')
+@click.command('shell', short_help='Run a Girder shell.', help='Run an interactive Girder shell '
+               'or a script in the Girder environment.')
 @click.option('--plugins', default=None, help='Comma separated list of plugins to import.')
-def main(plugins):
+@click.argument('script', type=click.Path(exists=True, dir_okay=False), required=False)
```

Comment:
If you want, this could be `click.File` which would pass an open file handle.

Reply:
I did that at first, but it turns out we need the actual path inside the function.